docs/submit-opentargets-batch.md

```diff
@@ -81,27 +95,37 @@ cd ${CODE_ROOT} && ${BSUB_CMDLINE} \
   -j clinvar-xml-parser/src/main/java
 ```
 
-A new Java package will be generated in the directory `clinvar-xml-parser/src/main/java/uk/ac/ebi/eva/clinvar/model`. With each schema version change, test data must be updated as well. See details in [Build instructions](build.md#regenerating-test-data). Create a pull request to merge this code into the main repository. It must contain both the updated schema version as well as the new test data. After a schema update, you'll also need to rebuild Java parser (see [Build instructions](build.md#building-java-clinvar-parser)).
+A new Java package will be generated in the directory `clinvar-xml-parser/src/main/java/uk/ac/ebi/eva/clinvar/model`. You will need to rebuild the parser; see [Build instructions](build.md#building-java-clinvar-parser).
+
+Update the test data. All the test does (for the moment) is checking that parsing 10 records from the XML will (1) not crash and (2) provide 10 records after parsing. So to regenerate test data, we just have to extract any 10 records (can just be the first 10 records) from the ClinVar XML file:
+```bash
+zcat ${BATCH_ROOT}/clinvar/ClinVarFullRelease_${CLINVAR_RELEASE}.xml.gz \
+  | awk 'BEGIN {RS="</ClinVarSet>\n\n"; ORS=RS} {print} NR==10 {exit}' \
+  > ${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml
+echo "</ReleaseSet>" >> ${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml
+gzip -c \
+  <${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml \
+  >${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml.gz
+```
+
+Eyeball input & output files to ensure that the ClinVar format has not changed sufficiently enough to render this snippet invalid.
+
+On the cluster, check out a new Git branch, commit all resulting file changes (which will include parser classes and test data), and push them to GitHub (you will need to enter your user name and either a password or an authentication token). Create a pull request to merge this code into the main repository.
 
-### 1.3. Convert ClinVar files
+### 1.3. Convert and filter ClinVar files
 Here we transform ClinVar's XML file into a JSON file which can be parsed by the downstream tools, using an XML parser which we (if necessary) updated during the previous step.
 
+Resulting Clinvar JSON (`clinvar.json.gz`) is then filtered, extracting only records with allowed levels of clinical significance (as provided by ClinVar). For example, this step filters out records where the clinical significance is “Benign”, meaning that the variant *does not* contribute to a disease. Result is output to `clinvar.filtered.json.gz`.
+
 ```bash
-cd ${CODE_ROOT} && ${BSUB_CMDLINE} -n 8 -M 4G \
+cd ${CODE_ROOT} && \
+${BSUB_CMDLINE} -K -n 8 -M 16G \
   -o ${BATCH_ROOT}/logs/convert_clinvar_files.out \
   -e ${BATCH_ROOT}/logs/convert_clinvar_files.err \
   java -jar ${CODE_ROOT}/clinvar-xml-parser/target/clinvar-parser-1.0-SNAPSHOT-jar-with-dependencies.jar \
```

Comment:
We are providing a memory requirement of 16G at the bsub level. Shouldn't we also provide that for the [java command](https://mkyong.com/java/java-what-is-xms-and-xmx-parameter/) through **-Xmx**?

Reply:
Addressed in https://github.com/EBIvariation/eva-cttv-pipeline/pull/87/commits/357b3060abcd9506b26f4024c29fd7432307861a

Reply:
(I pass `-Xmx15G` on purpose so that Java doesn't get too close to the bsub limit)